features/received_command.feature

```diff
@@ -66,7 +66,7 @@ Feature:  received_command
     Given a deployed rule:
       """
       rule 'Execute rule when member of group receives any command' do
-        received_command AlarmModes.items
+        received_command AlarmModes.members
```

Comment:
Is this a breaking change or fixing a test case?

Reply:
note that https://github.com/boc-tothefuture/openhab-jruby/pull/279 will make this case behave as you think it would